app/survey_config/business_config.py

```diff
@@ -19,20 +19,16 @@ class BusinessSurveyConfig(
     def __post_init__(self):
         super().__post_init__()
 
+        self.base_url = self._stripped_base_url
```

Comment:
This needs to be called before calling `super().__post_init__()` as that makes use of base url. At the moment the links they generate are also broken. So tests should be updated to assert them as well.

```py
    def __post_init__(self):
        self.contact_us_url: str = f"{self.base_url}/contact-us/"
        self.cookie_settings_url: str = f"{self.base_url}/cookies/"
        self.privacy_and_data_protection_url: str = (
            f"{self.base_url}/privacy-and-data-protection/"
        )
 ```